Sources/ResultModel/AnswerResult.swift

```diff
@@ -51,11 +51,11 @@ public extension AnswerResult {
     }
 }
 
-public final class AnswerResultObject : SerializableResultData, AnswerResult, MultiplatformTimestamp {
+public final class AnswerResultObject : ResultData, AnswerResult, MultiplatformTimestamp {
```

Comment:
Note: This result model has a custom decoder and encoder so it doesn't use `@Serializable`